apps/docs/tests/e2e/homePage.cy.ts

```diff
@@ -0,0 +1,57 @@
+import { DetailsPageObject } from '../support/pageObject/DetailsPageObject.PageObject';
+import { HomePageObject } from '../support/pageObject/HomePageObject.PageObject';
+
+const homePage = () => new HomePageObject();
+const detailsPage = () => new DetailsPageObject();
+
+describe('HomePageTests', () => {
+  it('Vue Instalation content', () => {
+    cy.visit('/');
+    homePage().clickInstallForVue();
+    detailsPage().isFramework('Vue').hasHeaderText('installation', 'Installation');
+
+    cy.fixture<Record<string, string>[]>('installationExamplesVue.json').then((examples) => {
+      examples.forEach((example) => {
+        cy.get('button').contains(`${example.button}`).click();
+        detailsPage().hasTitle(example.framework, example.app);
+      });
+    });
+  });
+
+  it('React installation content', () => {
+    cy.visit('/');
+    homePage().clickInstallForReact();
+    detailsPage().isFramework('React').hasHeaderText('installation', 'Installation');
+
+    cy.fixture<Record<string, string>[]>('installationExamplesReact.json').then((examples) => {
+      examples.forEach((example) => {
+        cy.get('button').contains(`${example.button}`).click();
+        detailsPage().hasTitle(example.framework, example.app);
```

Comment:
I'm wondering if it's worth to create new pageObject instance on every loop. It would be better to create it once and assign to variable

Reply:
or is it needed after the button click?

Reply:
If we will create it once and assign to variable there might be a case where `get` or `find` is cached and element we required in certain time in test wont be our element, in this case it does not matter because we have `get` inside of this method with dynamic look up. That why I would prefer to have pattern that works even in edge cases, so we would not spend extra time on finding why assigning pageObject to variable cause unexpected issues. I remember we did spend a lot of time on this in previous company.

This whole test do test every each page from left column and check if it has correct title so we need to first click another link `accordion` then `button` and so forth.